src/main/java/uk/gov/hmcts/probate/service/exceptionrecord/ExceptionRecordService.java

```diff
@@ -114,4 +140,104 @@ public SuccessfulTransformationResponse createGrantOfRepresentationCaseFromExcep
             throw new OCRMappingException(e.getMessage());
         }
     }
+
+    public SuccessfulCaveatUpdateResponse updateCaveatCaseFromExceptionRecord(
```

Comment:
The class updateCaveatCaseFromExceptionRecord needs a unit test.

Reply:
done on my commit - needs checking